kolibri/plugins/learn/assets/src/views/exam-page/index.vue

```diff
@@ -118,7 +118,7 @@
       previousQuestion: 'Previous question',
       nextQuestion: 'Next question',
       cancel: 'Cancel',
-      areYouSure: 'Are you sure you want to submit your exam?',
```

Comment:
Since this is a **learner facing query**, this is the only instance where I would actually maintain the question, maybe even add `(you can't change your answers after you submit)` at the end.

Reply:
updated let me know if you still prefer having 'Are you sure you...'

Reply:
Agreed, this seems like a good compromise! 👍